src/containers/Wise.jsx

```diff
@@ -0,0 +1,316 @@
+import React, {Component, useState} from "react";
+import {Container, Row, Col, Button, Accordion, Card} from "react-bootstrap";
+import url from "url";
+import PropTypes from "prop-types";
+import Footer from "../components/Footer.jsx";
+import PromptActionButtons from "../components/PromptActionButtons.jsx";
+import "./styles/Wise.scss";
+
+const DISCORD_LINK = "https://discord.gg/MRQq7H";
+const MAILING_LIST_FACEBOOK_LINK = "https://forms.gle/taTaauMpc193CTSFA";
+const FEEDBACK_FORM_LINK = "https://forms.gle/V6LNiG9zpqriavEh6";
+
+const girlsImage = url.resolve(
+  process.env.PUBLIC_URL,
+  "/illustrations/wise-girls.png",
+);
+const computerImage = url.resolve(
+  process.env.PUBLIC_URL,
+  "/illustrations/computer.png",
+);
+const wiseLogo = url.resolve(process.env.PUBLIC_URL, "/logos/wise-logo.png");
+
+const ExternalLink = ({href, children, ...restProps}) => (
+  <a href={href} target="_blank" rel="noopener noreferrer" {...restProps}>
+    {children}
+  </a>
+);
+ExternalLink.propTypes = {
+  href: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+const FAQS = [
+  {
+    question: "Who does WiSE support?",
+    answer: (
+      <p>
+        WiSE recognizes women, as well as those whose gender identity is neither
+        male nor female, including gender-fluid, non-binary, agender and more.
+        WiSE advocates for representation of all genders in software
+        engineering, and welcomes people with under-represented gender
+        identities to all of our events. Additionally, some of WiSE’s events are
+        open to all genders.
+      </p>
+    ),
+  },
+  {
+    question: "What does “women-only” mean?",
+    answer: (
+      <p>
+        Our women only events are open to cis and trans women, and those whose
+        gender identity is neither male nor female, including gender-fluid,
+        non-binary, agender and more. We welcome people with under-represented
+        gender identities to our women-only events.
+      </p>
+    ),
+  },
+  {
+    question: "What events do you run each term?",
+    answer: (
+      <p>
+        Events vary from term to term, but in the past we have run a brunch,
+        various mentoring activities, several casual board game nights, and a
+        “How to get an awesome co-op” workshop. Check out the events page WiSE
+        calendar for more information about our upcoming events!
```

Comment:
Is there something separate on the events page for WiSE?

Reply:
Updated to add an event calendar on the page, just need to add env vars (assuming you would have to do that? not sure)

Reply:
Curious which env vars do you need to add? But feel free to add them in :)

Reply:
For the wise calendar! Can you add them for me pls 😄  I'll slack them to you